dev/WindowsAppRuntime_DLL/packages.config

```diff
@@ -5,6 +5,6 @@
   <package id="Microsoft.SourceLink.GitHub" version="1.1.1" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.230706.1" targetFramework="native" />
   <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.240803.1" targetFramework="native" />
-  <package id="Microsoft.FrameworkUdk" version="1.8.0-CI-26107.1735.250116-1144.1" targetFramework="native" />
```

Comment:
I don't think we can add these here because Maestro has to update them using version.details.xml. I would suggest using an intermediary csproj like the one we have in WinUI